src/core/isFalse.js

```diff
@@ -0,0 +1,9 @@
+/** @license ISC License (c) copyright 2019 original and current authors */
```

Comment:
I have them in core because I was copying what we've done with predicates elsewhere but I see no reason that they need to other than convention. I'm not on my computer at the moment but I could look for where we could use this around the code, I'm sure there is a few places it could be used